analyzers/src/SonarAnalyzer.CSharp/SonarAnalyzer.CSharp.csproj

```diff
@@ -61,6 +61,17 @@
     </ItemGroup>
     <Copy SourceFiles="@(BinariesToCopy)" DestinationFolder="$(BinariesFolder)" />
   </Target>
-  <Import Project="..\SonarAnalyzer.Shared\SonarAnalyzer.Shared.projitems" Label="Shared" />
 
+  <Target Name="CleanBinaries" AfterTargets="Clean">
+    <ItemGroup>
+      <BinariesToDelete Include="$(BinariesFolder)\Google.Protobuf.dll" />
+      <BinariesToDelete Include="$(BinariesFolder)\SonarAnalyzer.dll" />
+      <BinariesToDelete Include="$(BinariesFolder)\SonarAnalyzer.CFG.dll" />
```

Comment:
This will collide with VB and doesn't solve the problem of deleting files that should no longer be there. We'll need to create subdirs per analyzer if you want to split this.

The deletion needs to solve the problem of `SonarAnalyzer.dll` was renamed to `SonarAnalyzer.Something.dll` but local packaging is still, by accident, or funky checked out commit, picking up the old file.

Reply:
If we change the file structure we need to update the consumers. Is it only the java plugin?

Reply:
There is also [this solution](https://stackoverflow.com/a/77933410) but it requires a lot more scaffolding. On the other hand, it would probably the cleanest.

Reply:
Java plugin, NuSpec files, .NET ITs AFAICT. Search for the well-known filenames in the repo should do the trick of finding what I missed

Reply:
Such "helper projects" will cause more confusion and might introduce some troublesome dependencies.